UPGRADE_GUIDE.md

```diff
@@ -45,6 +47,23 @@ public ComponentType[] MinimumComponentTypes { get; } =
 
 > You will need to add `using Unity.Entities;` to the top of the file and reference it in the assembly that your custom GameObject creator is in.
 
+#### `void Register()` method
+
+Use this method to register entity types and the method required to create GameObjects of that type.
+
+For example, the FPS project does the following:
+
+```csharp
+public void Register(Dictionary<string, EntityTypeRegistration> entityTypeRegistrations)
+{
+    entityTypeRegistrations.Add(PlayerEntityType, new EntityTypeRegistration(CreatePlayerGameObject,
+        typeof(OwningWorker.Component),
+        typeof(ServerMovement.Component)));
+}
+```
+
+You can optionally leave the implementation of this method blank, as the `GameObjectInitializationSystem` will use the creator's `OnEntityCreated` method by default.
```

Comment:
can you add this bit to the readme doc that you updated?  I think this is really useful when reading through that guide as well

Reply:
Already done :)